### PR TITLE
[SEASK-68] Quick Start -> Automated VM Set Up - Wallpaper

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: packer/scripts/server-setup.ps1 [hunk 1]
```diff
@@ -171,10 +179,25 @@ function Install-PreRequisites() {
...
+    
     Stop-Transcript
 }
 
+function Set-WallPaper {
+    Write-Output "Downloading wallpaper image"
+    $url = "https://edfidata.s3-us-west-2.amazonaws.com/Starter+Kits/images/EdFiQuickStartBackground.png"
+    $downloadedFile = Get-FileFromInternet $url
+	
+    Copy-Item -Path $downloadedFile -Destination "C:/"
```
Comment (vimayya) [inline packer/scripts/server-setup.ps1]: @raghuramakrishna copy it under user directory or under C:\Ed-Fi-Starter-Kit

**Review by jamessmckay — CHANGES_REQUESTED** [review]
@raghuramakrishna please update the destination per the request.

File: packer/scripts/server-setup.ps1 [hunk 2]
```diff
@@ -171,10 +179,25 @@ function Install-PreRequisites() {
...
+	
+    Set-ItemProperty -path "HKCU:Control Panel\Desktop" -name WallPaper -value "C:/EdFiQuickStartBackground.png"
+    Set-ItemProperty -path "HKCU:\Control Panel\Desktop" -name WallpaperStyle -value "6"
```
Comment (vimayya) [inline packer/scripts/server-setup.ps1]: @raghuramakrishna Choose fit as Center and not Stretch/Fit

File: packer/scripts/server-setup.ps1 (hunk 1 above)
Comment (raghuramakrishna) [reply packer/scripts/server-setup.ps1]: updated

File: packer/scripts/server-setup.ps1 (hunk 2 above)
Comment (raghuramakrishna) [reply packer/scripts/server-setup.ps1]: updated

**Review by jamessmckay — APPROVED** [review]

**Pull request merged**